app/components/external_links/psu_digital_collections_links_component.html.erb

```diff
@@ -6,8 +6,12 @@
                      class: 'mr-3') %>
 
   <ul class="external-links-psu-digital-collections">
-    <% psu_digital_collections_links.map do |item| %>
-      <li><i class="fas fa-external-link-alt" aria-hidden="true"></i> <%= item['prefix'] %><a href="<%= item['url'] %>"><%= item['text'] %></a><%= item['notes'] %></li>
+    <% links.map do |item| %>
+        <li>
+          <i class="fas fa-external-link-alt" aria-hidden="true"></i> <%= item['prefix'] %>
+          <%= link_to item['text'], item['url'] %>
+          <%= item['notes'] %>
+        </li>
```

Comment:
Use the ExternalLinks::LinkComponent here.

Reply:
ah yes, thanks!